packages/web/src/app/login/page.tsx

```diff
@@ -0,0 +1,87 @@
+}) {
+    return (
+        <div className="flex flex-col justify-center items-center h-screen">
+            <div className="flex flex-col items-center border p-16 rounded-lg gap-6">
+                <div>
+                    <Image
+                        src={logoDark}
+                        className="h-16 w-auto hidden dark:block"
+                        alt={"Sourcebot logo"}
+                        priority={true}
+                    />
+                    <Image
+                        src={logoLight}
+                        className="h-16 w-auto block dark:hidden"
+                        alt={"Sourcebot logo"}
+                        priority={true}
+                    />
+                </div>
+                {
+                    Object.values(providerMap)
+                        .map((provider) => {
+                            if (provider.id === "github") {
+                                return {
+                                    provider,
+                                    logo: githubLogo,
+                                }
+                            }
+
+                            return { provider }
+                        })
+                        .map(({ provider, logo }) => (
+                            <form
+                                key={provider.id}
+                                action={async () => {
+                                    "use server"
+                                    try {
+                                        await signIn(provider.id, {
+                                            redirectTo: props.searchParams?.callbackUrl ?? "",
+                                        })
+                                    } catch (error) {
+                                        // Signin can fail for a number of reasons, such as the user
+                                        // not existing, or the user not having the correct role.
+                                        // In some cases, you may want to redirect to a custom error
+                                        if (error instanceof AuthError) {
+                                            return redirect(`${SIGNIN_ERROR_URL}?error=${error.type}`)
+                                        }
+
+                                        // Otherwise if a redirects happens Next.js can handle it
+                                        // so you can just re-thrown the error and let Next.js handle it.
+                                        // Docs:
+                                        // https://nextjs.org/docs/app/api-reference/functions/redirect#server-component
+                                        throw error
+                                    }
+                                }}
+                            >
+                                <Button
+                                    type="submit"
+                                >
+                                    {logo && (
```

Comment:
nit: github logo here is white against white button (im using dark mode, not sure if its better in light mode)